src/react-valence-ui-iframe.js

```diff
@@ -44,6 +48,28 @@ var ResizingIframe = React.createClass({
 			}
 		}
 	},
+	updateNavbarStyle: function() {
+		// Hide the navbar and minibar from within the iframe if it is rendered
+		var iframe = React.findDOMNode(this.refs.iframe);
+		// Only update if the frame location has changed
+		if (this.state.iframeLocation !== iframe.contentWindow.location.href) {
```

Comment:
That's a good point. It should check whether it is cross domain before checking the location here to make sure it has access to the contentWindow.